dts/bindings/base/pm.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# Properties for Power Management (PM)
+
+properties:
+    wakeup-source:
+      required: false
+      type: boolean
+      description: |
+        Property to identify that a device can be used as wake up source.
+
+        When this property is provided a specific flag is set into the
+        device that tells the system that the device is capable of
+        wake up the system.
+
+        Wake up capable devices are enabled (interruptions will wake up
+        the system) by default but they can be disabled at runtime if necessary.
```

Comment:
If `pm.yml` is not included by default, devices that use `wakeup-source` need to include it. Not sure what is better, but if not included by `base.yaml` docs should mention it.

Reply:
This should be included in `base.yaml`, at least for now IMHO.